src/main/java/edu/kis/powp/jobs2d/drivers/visitor/DriversCountVisitor.java

```diff
@@ -6,41 +6,27 @@
 import edu.kis.powp.jobs2d.drivers.logger.LoggerDriver;
 
 import java.util.List;
-import java.util.stream.Collectors;
 
 public class DriversCountVisitor implements IDriverVisitor {
-    private long count = 0;
-
-    public long getDriversCount() {
```

Comment:
I think that this field and the getter are necessary also in stateless implementation.